### PR TITLE
Update Shortcodes Style page

### PR DESCRIPTION
**Pull request by IanMadd**
Signed-off-by: Ian Maddaus <Ian.Maddaus@progress.com>

## Description

Some improvements to clarify the difference between markdown and html shortcodes.

## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/style_reuse.md [hunk 1]
```diff
@@ -13,22 +13,59 @@ product = []
...
-To include a shortcode in a Markdown file, wrap the name of the shortcode file, without the file type suffix, in double curly braces and percent characters. For example, if you wanted to add the `chef.md` shortcode to a page, add the following text to the Markdown page:
+To include a Markdown shortcode in a page, wrap the name of the shortcode file, without the file type suffix, in double curly braces and percent characters. For example, if you wanted to add the [`chef.md`](https://github.com/chef/chef-web-docs/blob/master/themes/docs-new/layouts/shortcodes/chef.md) shortcode to a page, add the following text to the Markdown page:
 
 ```go
 {{%/* chef */%}}
 ```
 
+### HTML Shortcodes
+
+To include an HTML shortcode in a page, wrap the name of the shortcode file, without the file type suffix, in double curly braces and less than and greater than characters. For example, add the following text to a page if you wanted to add the [`chef_automate_mark.html`](https://github.com/chef/chef-web-docs/blob/master/themes/docs-new/layouts/shortcodes/chef_automate_mark.html) shortcode:
```
Comment (kagarmoe) [inline content/style_reuse.md]:
```suggestion
To include an HTML shortcode in a page, wrap the name of the shortcode file, without the file type suffix, in between double curly braces and angle brackets, `{{< SHORTCODE >}}`. For example, add the following text to a page if you wanted to add the [`chef_automate_mark.html`](https://github.com/chef/chef-web-docs/blob/master/themes/docs-new/layouts/shortcodes/chef_automate_mark.html) shortcode:
```

File: content/style_reuse.md [hunk 2]
```diff
@@ -13,22 +13,59 @@ product = []
...
+
+{{< note >}}
+If you add a **Markdown** shortcode to a page using **HTML** shortcode delimiters, Hugo will assume that the text is already formatted in HTML and won't run the shortcode file through its Markdown processor, leaving the bare Markdown in the HTML page output.
+{{< /note >}}
 
-### Adding a Shortcode to a Page
+### Markdown Shortcodes
 
-To include a shortcode in a Markdown file, wrap the name of the shortcode file, without the file type suffix, in double curly braces and percent characters. For example, if you wanted to add the `chef.md` shortcode to a page, add the following text to the Markdown page:
+To include a Markdown shortcode in a page, wrap the name of the shortcode file, without the file type suffix, in double curly braces and percent characters. For example, if you wanted to add the [`chef.md`](https://github.com/chef/chef-web-docs/blob/master/themes/docs-new/layouts/shortcodes/chef.md) shortcode to a page, add the following text to the Markdown page:
```
Comment (kagarmoe) [inline content/style_reuse.md]:
```suggestion
To include a Markdown shortcode in a page, wrap the name of the shortcode file, without the file type suffix, in between double curly braces and percent characters, `{{% SHORTCODE %}}`. For example, if you wanted to add the [`chef.md`](https://github.com/chef/chef-web-docs/blob/master/themes/docs-new/layouts/shortcodes/chef.md) shortcode to a page, add the following text to the Markdown page:
```

File: content/style_reuse.md [hunk 3]
```diff
@@ -13,22 +13,59 @@ product = []
...
 
 ```go
 {{%/* chef */%}}
```
Comment (kagarmoe) [inline content/style_reuse.md:35]: I don't understand the `/* ... */` pair in this example.

File: content/style_reuse.md [hunk 4]
```diff
@@ -13,22 +13,59 @@ product = []
...
+
+The [Fontawesome Shortcode](#fontawesome-shortcode) accepts named parameters. For example, it accepts a class value which is added like this: `{{</* fontawesome class="fas fa-ellipsis-h" */>}}`
+
+See the [Fontawesome Shortcode](#fontawesome-shortcode) section for more examples.
+
+### Nested Content
+
+We have some shortcodes that nest around Markdown content that is included in the text of a page. Those shortcodes are all written in HTML. Note the slash `/` before the name of the closing shortcode.
+
+
```
Comment (kagarmoe) [inline content/style_reuse.md]:
```suggestion
```

File: content/style_reuse.md [hunk 5]
```diff
@@ -13,22 +13,59 @@ product = []
...
+```md
+{{</* shortcode_name */>}}
```
Comment (kagarmoe) [inline content/style_reuse.md:59]: `/*`?

File: content/style_reuse.md [hunk 6]
```diff
@@ -13,22 +13,59 @@ product = []
...
+
+Some Markdown text.
+
+{{</* /shortcode_name */>}}
```
Comment (kagarmoe) [inline content/style_reuse.md:63]: `/*`?

File: content/style_reuse.md [hunk 7]
```diff
@@ -261,7 +298,7 @@ and `panel-id`/`panel-link` values must be unique on the page.
     {{</* /foundation_tabs_panels */>}}
```
Comment (kagarmoe) [inline content/style_reuse.md:297]: `/*`?

File: content/style_reuse.md [hunk 8]
```diff
@@ -261,7 +298,7 @@ and `panel-id`/`panel-link` values must be unique on the page.
...
 
 
-## Fontawesome
+## Fontawesome Shortcode
 
 The Fontawesome shortcode will display any free [Font Awesome icon](https://fontawesome.com/icons) in a page.
```
Comment (kagarmoe) [inline content/style_reuse.md:302]: How do we resize or style Fontawesome icons?

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Some questions and a few changes.

File: content/style_reuse.md (hunk 3 above)
Comment (IanMadd) [reply content/style_reuse.md:35]: It prevents hugo from processing it as a shortcode.

File: content/style_reuse.md (hunk 8 above)
Comment (IanMadd) [reply content/style_reuse.md:302]: That comes after line 303.

File: content/style_reuse.md (hunk 5 above)
Comment (IanMadd) [reply content/style_reuse.md:59]: Same as above. It prevents hugo from processing it as a shortcode.

File: content/style_reuse.md (hunk 6 above)
Comment (IanMadd) [reply content/style_reuse.md:63]: Same as above.

**Review by kagarmoe — APPROVED** [review]
Thanks!

**Pull request merged**